Example/ios/ReanimatedExample/AppDelegate.m

```diff
@@ -1,54 +1,32 @@
 #import "AppDelegate.h"
-
 #import <React/RCTBridge.h>
 #import <React/RCTBundleURLProvider.h>
 #import <React/RCTRootView.h>
-
-#import <React/RCTImageLoader.h>
-#import <React/RCTLocalAssetImageLoader.h>
-#import <React/RCTGIFImageDecoder.h>
-#import <React/RCTNetworking.h>
-#import <React/RCTHTTPRequestHandler.h>
-#import <React/RCTDataRequestHandler.h>
-#import <React/RCTFileRequestHandler.h>
-#import "MBFingerTipWindow.h"
```

Comment:
Can we bring this line back?

Reply:
Just `#import "MBFingerTipWindow.h"` or all of the imports?

Reply:
Update according to documentation `nttld/setup-ndk@v1` action caching NDK by default:
`https://github.com/nttld/setup-ndk` - "This action sets up an Android NDK environment by downloading and caching a version of the NDK and adding it to the PATH"